src/databricks/labs/ucx/source_code/site_packages.py

```diff
@@ -21,7 +22,7 @@ def __init__(
         self,
         site_packages: SitePackages,
         file_loader: FileLoader,
-        syspath_provider: SysPathProvider,
+        syspath_provider: PathLookup,
```

Comment:
also rename the `syspath_provider` references to `path_lookup`

Reply:
done